src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerMiddleware.cs

```diff
@@ -89,7 +89,11 @@ static async Task Awaited(ExceptionHandlerMiddleware middleware, HttpContext con
 
         private async Task HandleException(HttpContext context, ExceptionDispatchInfo edi)
         {
-            _logger.UnhandledException(edi.SourceException);
+            if (!_options.SuppressExceptionLog)
```

Comment:
@davidfowl are we really doing this? I can't think of anywhere else in the framework that you can control logging on a component via its options. Is this case special because if you handle the error then it should no longer be logged at the error level? If this exception is considered handled then maybe we should instead downgrade it to info or debug.

Reply:
@jez9999 would downgrading the log level used here help in your scenario?

Reply:
Yea I thought we resolved the issue tbh. Not sure why were fixing this when it's possible to suppress the log. I guess this is more discoverable but as you say, we don't do this anywhere else

Reply:
Dunno, I think the "None" logging option was ok, it's just a shame Serilog doesn't support it.  Not ASP.NET Core's fault though.  However Serilog filters can be used instead.

Reply:
Ok, closing this PR and issue as resolved then.